src/input.css

```
@@ -129,21 +129,22 @@
 
   /* Mobile menu toggle button */
   .nav-mobile-toggle {
-    @apply inline-flex items-center justify-center p-2 rounded-md;
+    @apply inline-flex size-11 items-center justify-center rounded-md;
     color: var(--text-muted);
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)